test/app_test_case.py

```diff
@@ -108,3 +162,54 @@ def tearDown(self):
         self.server_thread.join(timeout=10)
         if self.server_thread.is_alive():
             self.fail('Thread is still alive after joining')
+
+    def set_up_fake_responses(self):
+        # This requires to be invoked in a method within the context of
+        # ``responses.activate`` or ``responses.RequestsMock()``.
```

Comment:
Consider making this method a decorator that calls `responses.activate` so as to enforce the dependency.

Also take a look at `test/retorts.py`. It works around a problem that we may also run into again when once we add auth to the cart endpoints AND run tests against those.

Instead of creating another decorator, the best approach may be to extend `ResponsesHelper` in `retorts.py` and to override the `__enter__` method in order to set up the mocked auth endpoint responses. 

If the decorator approach works for now, I'd be happy with that, too.

Also, prefer the term `mock` over `fake`.

Reply:
The setup for mock responses is moved to `AuthResponsesHelper` in `test/retorts.py` and the test case is split up into `LocalAppTestCase` and `AuthLocalAppTestCase` (still in the same file).